MovieGallery/Pages/Search.cshtml

```diff
@@ -105,63 +105,76 @@
 
 else
 {
-    if (Model.SearchCompleted)
+    if (Model.CorrectYear)
     {
-        <table class="table table-striped table-light">
-            <thead class="thead-dark">
-                <tr>
-                    <th>Title</th>
-                    <th>Crew</th>
-                    <th>IMDb Rating</th>
-                    @*<th>image</th>*@
-                    <th>Type</th>
-                </tr>
-            </thead>
-            <tbody>
-                @foreach (var movie in Model.MoviesResult.Items)
-                {
-
+        if (Model.SearchCompleted)
+        {
+            <table class="table table-striped table-light">
+                <thead class="thead-dark">
                     <tr>
-                        <td>@movie.FullTitle</td>
-                        <td>@movie.Crew</td>
-                        <td>@movie.ImDbRating</td>
-                        <td>@movie.Type</td>
-                        @*<td>@movie.Image</td>*@
+                        <th>Title</th>
+                        <th>Crew</th>
+                        <th>IMDb Rating</th>
+                        @*<th>image</th>*@
+                        <th>Type</th>
```

Comment:
This is inside a loop, so td makes more sense.  th is only for table headings (the first row).